File: 백준/Bronze/10807. 개수 세기/개수 세기.java
import java.io.*;
public class Main {

    public static void main(String[] args) throws IOException {

        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));

        int N = Integer.parseInt(br.readLine());
        String s=br.readLine();
        String v = br.readLine();
        String arr[]=s.split(" ");
        int count=0;
        for(int i = 0; i<N; i++) {
            if(arr[i].equals(v)){
            count++;
            }
        }
        bw.write(count+"");
        bw.flush();
        bw.close();
    }
}

File: 백준/Bronze/10807. 개수 세기/개수 세기.cc
#include <iostream>
#include <algorithm>
using namespace std;

int main() {
    int n,v;
    cin >> n;
    int arr[100];
    for (int i = 0; i < n;i++) {
        cin >> arr[i];
    }
    cin >> v;
    cout<<count(arr, arr + n, v);

    return 0;
}
